packages/css-modules-flow-types-loader/index.js

```diff
@@ -3,12 +3,52 @@
 import fs from 'fs';
 import printFlowDefinition from 'css-modules-flow-types-printer';
 
+/**
+ * @desc
+ * Parse the string representation of css-modules ICSS source code
+ * and output local classNames if any were found
+ *
+ * @param {String} content
+ * @return {Object<String, String>}
+ */
 function getTokens(content) {
-  const cssTokens = content.replace(/\s/g, '').match('exports.locals=(.*);');
-  if (cssTokens) {
-    return JSON.parse(cssTokens[1]);
+  // Only `locals` export is desired
+  const locals = content.match(/exports\.locals = ([\s\S]*);/);
+
+  if (!locals) return {};
+
+  /**
+   * Collect object keys by parsing the string
```

Comment:
IMO if a code block inside a function needs this much documentation, then it clearly needs its own function.

Reply:
Ditto… getting back to old code, one might be inclined to "optimize" away seemingly extraneous code.

Now this is covered by tests, so no harm should be done, but giving away the rationale right from the outset seems like a good habit ...

Reply:
> be inclined to "optimize" away seemingly extraneous code.

That is why I prefer all cases being covered by tests. Comments don't guarantee anything... And comments might get outdated over time. : )

> but giving away the rationale right from the outset seems like a good habit 

Hence making this a separate function with a good name. That should be enough documentation IMO.